task/sbom-json-check/0.1/sbom-json-check.yaml

```diff
@@ -55,10 +57,12 @@ spec:
       FAIL_RESULTS="$(cat fail_result.txt)"
       if [[ -z $FAIL_RESULTS ]]
       then
-        HACBS_TEST_OUTPUT=$(make_result_json -r "SUCCESS" -s 1)
+        note="Task $(context.task.name) completed: Check field `result` for the json check result."
```

Comment:
SR: note="Task $(context.task.name) completed: Check `result` for the JSON check result."

Reply:
Fixed